CLTokenInputView/CLTokenInputView/CLTokenInputView.h

```diff
@@ -22,6 +22,11 @@
 #define CL_GENERIC_MUTABLE_SET(type) NSMutableSet
 #endif
 
+typedef NS_ENUM(NSInteger, CLTokenStyle) {
+    CLTokenStyleText,
```

Comment:
Could you add screenshots of what both styles look like?

Reply:
uploaded new pill style and existing text style

Reply:
Saw them. They look good!